src/osg/Texture2DArray.cpp

```diff
@@ -336,53 +336,42 @@ void Texture2DArray::apply(State& state) const
         // compute the internal texture format, this set the _internalFormat to an appropriate value.
         computeInternalFormat();
 
-        GLenum texStorageSizedInternalFormat = extensions->isTextureStorageEnabled && (_borderWidth==0) ? selectSizedInternalFormat(_images[0].get()) : 0;
```

Comment:
I think this whole file can go back to the 3.6 version now that selectSizedInternalFormat is going back into Texture.cpp